src/index.js

```diff
@@ -90,12 +90,40 @@ class BodyBuilder {
    * @returns {BodyBuilder} Builder class.
    */
   sort(field, direction = 'asc') {
-    this._body.sort = {
-      [field]: {
-        order: direction
+    this._body.sort = this._body.sort || [];
+    var self = this;
```

Comment:
I think you won't need to stash a reference to this if you use arrow functions on L104 and L105